eth/tracers/logger/logger.go

```diff
@@ -132,6 +134,17 @@ func NewStructLogger(cfg *Config) *StructLogger {
 	return logger
 }
 
+func NewStructLoggerStream(cfg *Config, stream chan interface{}) *StructLogger {
```

Comment:
Can't you use the `NewJSONLogger` for this instead? It is already meant for streaming, but it now streams `string` via it's internal encoder. Maybe the two versions can be made to play nice, somehow

Reply:
Ah I had actually never used the JSON logger, good to know its meant for streaming.